todomvrx/build.gradle

```diff
@@ -33,7 +33,6 @@ dependencies {
     kapt "com.airbnb.android:epoxy-processor:$epoxyVersion"
     implementation 'io.reactivex.rxjava2:rxandroid:2.1.1'
     implementation "io.reactivex.rxjava2:rxjava:2.2.8"
-    implementation "org.koin:koin-android-architecture:$koinVersion"
```

Comment:
Does this need to add the implementation of koin-android? or no?

Reply:
What's the point of not using Android specific Koin implementation in an Android app?

Reply:
I was just wondering if something needed to replace the deleted line in the Gradle file. If not, then no big deal 🙂

Reply:
I think the Koin folks stopped shipping the `org.koin:koin-android-architecture` artifact, which is why I removed it.

Reply:
I believe this is just a copy paste. `todomvrx` project actually does not even use Koin. I'm sure there are more undeeded dependencies here.

Reply:
Removed bunch more in my previous PR https://github.com/airbnb/MvRx/pull/255/files#diff-9b2ebc432ba289ec4a9b8f3feb124159L48